index.d.ts

```diff
@@ -325,6 +332,7 @@ declare namespace contextMenu {
 		- `showCopyVideoAddress`
 		- `showSaveVideo`
 		- `showSaveVideoAs`
+		- `showCopyLink`
```

Comment:
This needs to be done in the readme too.

Reply:
Oops, sorry fixed it now.